src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,95 @@
+import cn from 'classnames';
+import React, { useState } from 'react';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const numberOfPages: number = Math.ceil(total / perPage);
+
+  const [activePage, setActivePage] = useState('1');
+
+  return (
+    <ul className="pagination">
+      <li
+        className={cn('page-item', {
+          disabled: currentPage === 1,
+          active: activePage === 'prev',
```

Comment:
The condition for the 'active' class seems incorrect. The 'prev' button should not be dependent on the `activePage` state, but rather on the `currentPage` to indicate if we are on the first page.